runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2872,6 +2872,7 @@ gcInitializeDefaults(J9JavaVM* vm)
 	extensions->setOmrVM(vm->omrVM);
 	vm->omrVM->_gcOmrVMExtensions = (void *)extensions;
 	vm->gcExtensions = vm->omrVM->_gcOmrVMExtensions;
+	vm->isIndexableDualHeaderShapeEnabled = false;
```

Comment:
`isIndexableDualHeaderShapeEnabled` is defined `BOOLEAN` so, should it be `FALSE` for consistency (and `TRUE` in parsing `-XXgc` option)?
`TRUE` is used for initialization of similar `BOOLEAN isDataAddrPresent` lower in the code

Reply:
Good catch, yes I will do this.